docker/quantum/README.md

```diff
@@ -45,3 +45,18 @@ A json description of the image and some of the options may be obtained via:
 ```
 docker run quantum:1.0 -d
 ```
+
+Register Image to Run Notebook
+=================================
+
+The easiest way:
+- Use the Girder API to register the local image:
+  - Visit http://localhost:8080/api/v1
+  - [GET cluster ID](http://localhost:8080/api/v1#!/clusters/clusters_find)
+  - [Register the local image](http://localhost:8080/api/v1#!/images/images_create)
+    - `type`: <strong>docker</strong>
+    - `repository`: <strong>quantum</strong>
+    - `tag`: <strong>1.0</strong>
+    - `digest`: <strong>123</strong> (can be any value)
+    - `clusterId`: (from GET /clusters)
+    - `size`: <strong>2</strong> (can be any value)
```

Comment:
We should be able to do this via the UI @psavery? Also how do we get it registered automatically in the dev environment?

Reply:
Looks like we needed to add it to this [list](https://github.com/OpenChemistry/openchemistrypy/blob/master/taskflows/taskflows/container_list_taskflow.py#L32).

Reply:
@cjh1 Yeah, that's to register it automatically.

You can also manually register it via the girder endpoint at `localhost:8080/api/v1` under "images". If you do this, you can put whatever you want for the digest and the size, the digest is only used to see if the image has already been registered. The size isn't used for anything but display.

Reply:
@bnmajor Lets add `openchemistry/qiskit` to the list to be added automatically, then I think we can remove this block.

Reply:
We might also want to add it [here](https://github.com/OpenChemistry/mongochemclient/blob/a75f70f569960e1a4c3b0a6b464c927b324c36f9/src/components/image-manager/advanced-pull.js#L21-L24)